pkgs/development/libraries/libavif/thumbnailer

```diff
@@ -0,0 +1,5 @@
+[Thumbnailer Entry]
+TryExec=@bindir@/gdk-pixbuf-thumbnailer
+Exec=@bindir@/gdk-pixbuf-thumbnailer -s %s %u %o
```

Comment:
As discussed on [IRC](https://logs.nix.samueldr.com/nixos-gnome/2020-11-02#1604328446-1604328578;), you will need to generate a `loaders.cache` file and pass it to the program through environment.